Tests/Integration/Network/NetworkConfigFixtures.swift

```diff
@@ -10,7 +10,7 @@ import XCTest
 
 enum NetworkConfigFixtures {
     static let alphaDev = DynamicNetworkConfig.AlphaDevelopment.make()
-    static let network: NetworkPreset = .dynamic(alphaDev)
+    static let network: NetworkPreset = .testNet
```

Comment:
Does this affect CI?

Reply:
no effect on CI, and should be testNet as thats the most stable test network